src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -95,7 +92,29 @@ const defaultProps = {
 };
 
 class SidebarLinks extends React.Component {
-    shouldComponentUpdate(nextProps) {
+    static getRecentReports(props) {
+        const activeReportID = parseInt(props.currentlyViewedReportID, 10);
+        const sidebarOptions = getSidebarOptions(
+            props.reports,
+            props.personalDetails,
+            props.draftComments,
+            activeReportID,
+            props.priorityMode,
+            props.betas,
+        );
+        return sidebarOptions.recentReports;
+    }
+
+    static getUnreadReports(reportsObject) {
+        const reports = _.values(reportsObject || {});
```

Comment:
Then remove it here.

```suggestion
        const reports = _.values(reportsObject);
```